### PR TITLE
#237 Refactor isMobile

### LINKED ISSUE #237: isMobile 통합
# Summary 

현재 Button 컴포넌트에서 사용중이 isMobile 관련 코드를 통합하는 것이 좋을 것 같습니다

# Tasks 

- [x] tools에 isMobile.ts 추가
- [x] .chat 클래스 이름 변경
- [x] SubmitButton 제거

### PR DESCRIPTION
**Pull request by imYourChoi**
# Summary 

It closes #237

# Extra info 

- Needs more than 2 reviewers? n
- Needs more than 10 minutes for review? y
- Needs to execute in order to review? y

# Images 

# Further Work 

- Do something...

### CODE REVIEW

File: src/components/common/Button.tsx [hunk 1]
```diff
@@ -72,11 +78,11 @@ const Button = ({
...
   return (
     <div
-      onClick={onClick}
+      onClick={disabled ? undefined : onClick}
       style={{ ...style }}
-      onMouseEnter={() => setHover(!(isMobile().phone || isMobile().tablet))}
+      onMouseEnter={() => setHover(!isMobile)}
       onMouseLeave={() => setHoverClicked(false)}
-      onMouseDown={() => setClicked(true)}
+      onMouseDown={() => setClicked(!disabled && true)}
```
Comment (14KGun) [inline src/components/common/Button.tsx:85]: 조건문 `!disabled` 로 바꿔도 되지않나요?

File: src/App.css [hunk 2]
```diff
@@ -34,7 +34,7 @@ body::-webkit-scrollbar {
   display: none;
 }
 
-.chat {
+.chat-selectable {
```
Comment (14KGun) [inline src/App.css:37]: https://github.com/sparcs-kaist/taxi-front/blob/da645f29d8661a28c4363e2be9f429ccd8224237/src/components/Chatting/MessagesBody/ChatSet.jsx#L77

같이 바꿔주셔야할것같아요

**Review by 14KGun — COMMENTED** [review]
수고하셨습니다~~

File: src/components/common/Button.tsx (hunk 1 above)
Comment (imYourChoi) [reply src/components/common/Button.tsx:85]: 아 바보 같았네요

**Review by 14KGun — APPROVED** [review]
수고하셨습니당 ~~ 😁😁

**Pull request merged**